src/coreclr/pal/inc/pal.h

```diff
@@ -2469,6 +2469,33 @@ typedef struct _KNONVOLATILE_CONTEXT_POINTERS {
     //
 
 } KNONVOLATILE_CONTEXT_POINTERS, *PKNONVOLATILE_CONTEXT_POINTERS;
+#elif defined(HOST_WASM)
+#define CONTEXT_CONTROL 0
+#define CONTEXT_INTEGER 0
+#define CONTEXT_FLOATING_POINT 0
+#define CONTEXT_FULL 0
+
+#define CONTEXT_XSTATE 0
+
+#define CONTEXT_EXCEPTION_ACTIVE 0x8000000L
+#define CONTEXT_SERVICE_ACTIVE 0x10000000L
+#define CONTEXT_EXCEPTION_REQUEST 0x40000000L
+#define CONTEXT_EXCEPTION_REPORTING 0x80000000L
+
+typedef struct _CONTEXT {
+    ULONG ContextFlags;
+
+    ULONG   Dr0_PAL_Undefined;
+    ULONG   Dr1_PAL_Undefined;
+    ULONG   Dr2_PAL_Undefined;
+    ULONG   Dr3_PAL_Undefined;
+    ULONG   Dr6_PAL_Undefined;
+    ULONG   Dr7_PAL_Undefined;
```

Comment:
```suggestion
```
Are these dummy fields needed for anything?

Reply:
They are not, indeed.